app/views/components/forms/_logo_and_title.html.erb

```diff
@@ -1,14 +1,20 @@
-<%- if form.title.present? || form.logo.present? %>
-<h2 class="word-break fba-modal-title" id="<%= "fba-form-title-#{form.short_uuid}" %>">
-  <%- if form.logo.present? %>
-  <div class="margin-bottom-2 text-center">
-    <%- if form.display_header_logo %>
-      <%= image_tag(form.logo.tag.url, alt: "#{form.organization.name} banner", class: "form-header-logo") %>
-    <% elsif form.display_header_square_logo %>
-      <%= image_tag(form.logo.logo_square.url, alt: "#{form.organization.name} logo", class: "form-header-logo-square") %>
+<header>
+  <h1
```

Comment:
The switch from h2 to h1 puts too much space above the title in the new version of the modal:

<img width="674" alt="Screenshot 2024-08-20 at 5 41 51 PM" src="https://github.com/user-attachments/assets/ef3fef08-295d-40bf-a0d2-8cc56eb76056">

I was using [this css](https://github.com/GSA/touchpoints/blob/3795f1302345b4f8ac40df633c9a65e35f072397/app/views/components/widget/_widget.css.erb#L102) to eliminate the top margin here.

Reply:
-1rem